components/card/card.html

```diff
@@ -0,0 +1,311 @@
+              <span class="sale-price">Rs. 1599</span>
+              <span class="mrp-price">Rs. 2599</span>
+              <span class="card-discount">(38%)</span>
+            </div>
+            <div class="card-footer">
+              <div class="icon-button-container card-button">
+                <button class="btn btn-icon">
+                  <i class="fas fa-shopping-cart"></i>
+                  Add to Cart
+                </button>
+              </div>
+              <i class="far fa-heart fa-lg card-heart-icon"></i>
+            </div>
+          </div>
+        </div>
+
+        <!-- CODE: Card With Badge -->
+        <div class="code-example-container">
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28211%2C211%2C211%2C1%29&t=vscode&wt=none&l=htmlmixed&width=889.5&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=15px&ln=false&fl=1&fm=Hack&fs=14.5px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card-vertical%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-img%2522%253E%250A%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522..%252F..%252Fassets%252Fimages%252Fwatch.jpg%2522%2520alt%253D%2522%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522card-badge%2522%253ENEW%253C%252Fspan%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-title%2522%253E%250A%2520%2520%2520%2520%2520%253Ch4%253EApple%2520Watch%253C%252Fh4%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-description%2522%253E%250A%2520%2520%2520%2520%2520%253Cp%253EStarlight%2520Aluminium%2520Sports%2520Band%253C%252Fp%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-price%2522%253E%250A%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522sale-price%2522%253ERs.%25201599%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522mrp-price%2522%253ERs.%25202599%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522card-discount%2522%253E%2838%2525%29%253C%252Fspan%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-footer%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522icon-button-container%2520card-button%2522%253E%250A%2520%2520%2520%2520%2509%253Cbutton%2520class%253D%2522btn%2520btn-icon%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-shopping-cart%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520Add%2520to%2520Cart%250A%2520%2520%2520%2520%2520%2509%253C%252Fbutton%253E%250A%2520%2520%2509%253C%252Fdiv%253E%250A%2520%2520%253Ci%2520class%253D%2522far%2520fa-heart%2520fa-lg%2520card-heart-icon%2522%253E%253C%252Fi%253E%250A%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+        </div>
```

Comment:
all frames code are not visible properly,

Reply:
Frames code are "scrollable",  can you just check once again and suggest if there still problem